docs/datascience/overview.md

```diff
@@ -20,6 +20,16 @@ The VS Code Marketplace offers a family of extensions that provide a first-class
 
 <div class="marketplace-extensions-datascience-python"></div>
 
+### Build an extension for notebooks
+
+We welcome (and encourage you to!) build Visual Studio Code extensions for your favorite notebooks tools. After going through [Your First Extension](https://github.com/microsoft/vscode-docs/blob/main/api/get-started/your-first-extension.md) tutorial, check out some of the extensions that augments the notebooks development experience in VS Code:
```

Comment:
Can you change this link to (/api/get-started/your-first-extension.md)?

Also "extensions that augment the"